modin/pandas/series.py

```diff
@@ -717,7 +711,7 @@ def autocorr(self, lag=1):  # noqa: PR01, RT01, D200
         """
         return self.corr(self.shift(lag))
 
-    def between(self, left, right, inclusive="both"):  # noqa: PR01, RT01, D200
+    def _between(self, left, right, inclusive=True):  # noqa: PR01, RT01, D200
```

Comment:
We set all parameter in API layer to default values of old pandas version, am I right?

Reply:
I think we should _not_ be setting any value, in this case at least, as we pass it from the compat entry point